js/libs.js

```diff
@@ -307,36 +308,34 @@ Fliplet.Registry.set('notification-inbox:1.0:core', function (element, data) {
           $target.removeClass('fa-spin');
         }).catch(function (error) {
           $target.removeClass('fa-spin');
-          Fliplet.UI.toast.error(error, {
+          Fliplet.UI.Toast.error(error, {
             message: 'Notification refresh failed'
           });
         });
       });
   }
 
-  function init() {
+  function init(options) {
     moment.updateLocale('en', {
       calendar : {
         sameElse: 'MMMM Do YYYY'
       }
     });
 
+    options.clearNewCountOnUpdate = true;
+    options.startCheckingUpdates = true;
+
     // Prompt user to enable notification or subscribe for push notification in the background
     var pushWidget = Fliplet.Widget.get('PushNotifications');
     if (pushWidget) {
       pushWidget.ask();
     }
-
-    appNotifications = Fliplet.Widget.get('Notifications');
-    if (appNotifications) {
-      // Initializa Notifications app component
-      appNotifications.init({
-        clearNewCountOnUpdate: true,
-        startCheckingUpdates: true
-      });
-    }
   }
 
+  Fliplet.Hooks.on('afterNotificationsInit', function (instance) {
```

Comment:
Where is this hook run?

Reply:
https://github.com/Fliplet/fliplet-widget-notifications/pull/6/files#diff-5f43a4df2d7ae4cc0f4420b4410cc67eR15

Not sure why it didn't show up in the other PR, I have pushed the commit manually again.

Reply:
https://github.com/Fliplet/fliplet-widget-notifications/pull/6/commits/741a7b468165e1d9a2a667b4555ad9e4f4be2904

Reply:
It did, but apparently you removed it in this commit ☝️

Reply:
whoops, I didn't notice I accidentally deleted that line!